source/_components/broadlink.markdown

```diff
@@ -3,20 +3,174 @@ title: "Broadlink"
 description: "Instructions on how to integrate Broadlink within Home Assistant."
 logo: broadlink.png
 ha_category:
+  - Remote
   - Switch
   - Sensor
 ha_release: 0.35
 ha_iot_class: Local Polling
 redirect_from:
+ - /components/remote.broadlink/
```

Comment:
Please remove the `- /components/remote.broadlink/`

Reply:
Done.